package.json

```diff
@@ -8,50 +8,50 @@
     "url": "https://gitlab.com/antora/antora-ui-default.git"
   },
   "engines": {
-    "node": ">= 8.0.0"
+    "node": ">= 17.0.0"
   },
   "browserslist": [
     "last 2 versions"
   ],
   "devDependencies": {
     "@asciidoctor/core": "~2.2",
-    "autoprefixer": "~9.7",
+    "autoprefixer": "~10.4",
     "browser-pack-flat": "~3.4",
-    "browserify": "~16.5",
-    "cssnano": "~4.1",
+    "browserify": "~17.0",
+    "cssnano": "~5.1",
     "eslint": "~6.8",
```

Comment:
skipped all eslint deps stuff since it truly messes up all js files with 'new' formatting standards